dotCMS/src/main/java/com/dotcms/rest/api/v1/vtl/VTLResource.java

```diff
@@ -58,7 +66,7 @@ public VTLResource() {
 
     @GET
     @NoCache
-    @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
+    @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML, MediaType.TEXT_PLAIN})
     @Path("/{folder}/{path: .*}")
     public Response get(@Context final HttpServletRequest request, @Context final HttpServletResponse response,
```

Comment:
should the folderName be validated in case of null or empty values?

Reply:
@nollymar I will make sure to include this when writing the integration test for the resource on a separate ticket